src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -151,6 +152,58 @@ private static string Download(string baseGitPath, string dataFile)
             return dataFile;
         }
 
+        public static IDataView LoadAdultDataset(MLContext mlContext)
+        {
+            // Download the file
+            string dataFile = DownloadAdultDataset();
+
+            // Define the columns to read
+            var reader = mlContext.Data.CreateTextLoader(
+                columns: new[]
+                    {
+                        new TextLoader.Column("age", DataKind.R4, 0),
+                        new TextLoader.Column("workclass", DataKind.TX, 1),
+                        new TextLoader.Column("fnlwgt", DataKind.R4, 2),
+                        new TextLoader.Column("education", DataKind.TX, 3),
+                        new TextLoader.Column("education-num", DataKind.R4, 4),
+                        new TextLoader.Column("marital-status", DataKind.TX, 5),
+                        new TextLoader.Column("occupation", DataKind.TX, 6),
+                        new TextLoader.Column("relationship", DataKind.TX, 7),
+                        new TextLoader.Column("ethnicity", DataKind.TX, 8),
+                        new TextLoader.Column("sex", DataKind.TX, 9),
+                        new TextLoader.Column("capital-gain", DataKind.R4, 10),
+                        new TextLoader.Column("capital-loss", DataKind.R4, 11),
+                        new TextLoader.Column("hours-per-week", DataKind.R4, 12),
+                        new TextLoader.Column("native-country", DataKind.R4, 13),
+                        new TextLoader.Column("IsOver50K", DataKind.BL, 14),
+                    },
+                separatorChar: ',',
+                hasHeader: true
+            );
+
+            return reader.Read(dataFile);
+        }
+
+        public static IEstimator<ITransformer> DefaultBinaryPipeline(MLContext mlContext)
+        {
+            var pipeline =
+               // Convert categorical features to one-hot vectors
+               mlContext.Transforms.Categorical.OneHotEncoding("workclass")
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("education"))
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("marital-status"))
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("occupation"))
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("relationship"))
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("ethnicity"))
+               .Append(mlContext.Transforms.Categorical.OneHotEncoding("native-country"))
+               // Combine all features into one feature vector
+               .Append(mlContext.Transforms.Concatenate("Features", "workclass", "education", "marital-status",
+                   "occupation", "relationship", "ethnicity", "native-country", "age", "education-num",
+                   "capital-gain", "capital-loss", "hours-per-week"))
+               // Min-max normalized all the features
+               .Append(mlContext.Transforms.Normalize("Features"));
```

Comment:
>(mlContext.Transforms.Normalize("Features") [](start = 22, length = 43)

There is no need to normalize OneHotEncoded variables as they already range between 0-1. Here we should only  normalize numeric features.